README.md

```diff
@@ -3,12 +3,74 @@ rab
 
 Repository containing code for [Researchers@Brown](https://vivo.brown.edu/), built with the [VIVO/Vitro platform](https://github.com/vivo-project).
 
-###Setup
-
- * setup a database for use with Vitro: `create database vitro charset DEFAULT CHARACTER SET utf8 DEFAULT COLLATE utf8_general_ci`
- * git clone this repository
- * `git clone --branch maint-rel-1.7 https://github.com/vivo-project/Vitro.git`
- * make a vitro data directory
- * adjust build.properties and runtime.properties to match your machine
- * adjust rebuild.sh to match your matchine
- * run `sudo ./rebuild.sh`
+
+## Pre-requisites
+In order to compile and run an instance of VIVO you'll need MySQL,
+Tomcat, and Ant.
+
+You can install Tomcat and MySQL via 'apt-get' on Linux or via Homebrew on a Mac.
+
+You can get Ant from the Apache web site: http://ant.apache.org/bindownload.cgi,
+extract it, copy it to a folder under `/usr/local/bin/`,
+and add it to your path.
+
+```
+tar -xvzf apache-ant-1.9.7-bin.tar.gz
+mv apache-ant-1.9.7-bin/ /usr/local/bin/ant/
+# add /usr/local/bin/ant/bin to your PATH
+```
+
+## Setup
+Setup a database for use with Vitro
+
+```
+CREATE USER 'vivouser'@'localhost' IDENTIFIED BY 'password';
+
+GRANT ALL PRIVILEGES ON * . * TO 'vivouser'@'localhost';
+
+CREATE DATABASE vivodev DEFAULT CHARACTER SET utf8;
+```
+
+Get the code for this repository and for Vitro.
+
+```
+git clone git@github.com:Brown-University-Library/rab.git
+cd rab
+git clone --branch maint-rel-1.7 https://github.com/vivo-project/Vitro.git
```

Comment:
Since this documentation was originally created, we migrated to Vitro 1.8. The correct branch to use is maint-rel-1.8